exercises/kindergarten-garden/src/main/java/Plant.java

```diff
@@ -1,21 +0,0 @@
-public enum Plant {
```

Comment:
Yes, this should still be here. When we use enums we usually include this in the starter implementation regardless of the difficulty of the exercise. This is because they're not really part of the solution to the problem, they're more part of the problem specification if that makes sense :) Sorry for the confusion @redshirt4! :)